channels/us.m3u

```diff
@@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
 https://thegateway.app/YouToo/CueTones/playlist.m3u8
 #EXTINF:-1 tvg-id="YUYUTV.us" tvg-name="YUYU TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/Bbvtcd7.png" group-title="",YUYU TV
 https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/index.m3u8
```

Comment:
works

Reply:
Already available on the Indian playlist.